Sources/Views/SubscriptionConfirmation.swift

```diff
@@ -187,7 +224,31 @@ private func teamOwner(_ currentUser: User) -> Node {
               style(width(.px(24)) <> height(.px(24)))
             ]
           ),
-          span([.text(currentUser.displayName)])
+          span(
+            [`class`([Class.size.width100pct])],
+            [.text(currentUser.displayName)]
+          ),
+          a([
+            id("remove-yourself-button"),
+            `class`([
+              Class.cursor.pointer,
+              Class.pf.colors.fg.red,
+              Class.pf.colors.link.red,
+              Class.type.light,
+              Class.pf.type.body.small,
+              Class.pf.type.underlineLink
+              ]),
+            onclick("""
+var ownerRow = this.parentNode.parentNode
+ownerRow.parentNode.removeChild(ownerRow)
+
+var teamMember = document.getElementById("team-member-template-without-remove").content.cloneNode(true)
+var teamMembersContainer = document.getElementById("team-members")
+teamMembersContainer.insertBefore(teamMember, teamMembersContainer.firstChild)
+
+updateSeats()
+""")
+            ], [.raw("Remove&nbsp;yourself")])
```

Comment:
Maybe something else, like "Don't include yourself as a seat", "Don't include yourself", or "I don’t want a seat".
```suggestion
            ], [.raw("Don’t&nbsp;include&nbsp;yourself&nbsp;as&nbsp;a&nbsp;seat")])
```

Reply:
I guess it looks nicer and makes more sense when you add more rows: 
<img width="204" alt="image" src="https://user-images.githubusercontent.com/658/64388435-12331280-d00d-11e9-9ebb-b8cb0f06eab7.png">